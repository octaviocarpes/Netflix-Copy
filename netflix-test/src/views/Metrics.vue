<template>
  <h1>Metrics</h1>
</template>

<script>
export default {
  
}
</script>

<style lang="scss" scoped>

</style>
