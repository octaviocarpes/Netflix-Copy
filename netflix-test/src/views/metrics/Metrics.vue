<template>
  <div class="metrics-container">
    <Navbar />
    <h1>Metrics</h1>
  </div>
</template>

<script>
import Navbar from "@/components/navbar/Navbar";
export default {
  name: "Metrics",

  components: {
    Navbar
  }
};
</script>

<style lang="scss" scoped></style>
