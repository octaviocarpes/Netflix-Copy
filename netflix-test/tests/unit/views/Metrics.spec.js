import Vue from "vue";
import Metrics from "@/views/metrics/Metrics.vue";

describe("Metrics.spec.js", () => {
  let cmp, vm;

  beforeEach(() => {
    cmp = Vue.extend(Metrics);
    vm = new cmp({}).$mount();
  });

  it("has the expected html structure", () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
