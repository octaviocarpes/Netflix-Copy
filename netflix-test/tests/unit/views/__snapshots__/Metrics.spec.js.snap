// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Metrics.spec.js has the expected html structure 1`] = `
<div
  class="metrics-container"
>
  <nav>
    <div
      class="nav-bar-container"
    >
      <div
        class="nav-bar-content-left"
      >
        <div
          class="header-logo"
        >
          <h1>
            NIETFLIX
          </h1>
        </div>
         
        <div
          class="links"
        >
          <ul>
            <li
              id="movies-link"
            >
              <router-link
                to="movies"
              >
                Movies
              </router-link>
            </li>
             
            <li
              id="profile-link"
            >
              <router-link
                to="profile"
              >
                Profile
              </router-link>
            </li>
             
            <li
              id="metrics-link"
            >
              <router-link
                to="metrics"
              >
                Metrics
              </router-link>
            </li>
          </ul>
        </div>
      </div>
       
      <div
        class="nav-bar-content-right"
      >
        <div
          class="user"
        >
          <img
            alt="profile icon"
            src=""
          />
           
          <p>
            
          </p>
        </div>
         
        <p>
          Logout
        </p>
      </div>
    </div>
  </nav>
   
  <h1>
    Metrics
  </h1>
   
  <div
    class="metrics-content"
  >
    <div
      class="top-users"
    >
      <h1>
        Top Users
      </h1>
       
      <!---->
    </div>
     
    <div
      class="top-movies"
    >
      <h1>
        Top Movies
      </h1>
       
      <!---->
    </div>
     
    <div
      class="application-access"
    >
      <h1>
        Application Access
      </h1>
       
      <div
        class="access-chart-container"
      >
        <canvas
          id="access-chart"
        />
         
        <div
          class="access-data"
        >
          <h2>
            Logins:
          </h2>
           
          <h3>
            0
          </h3>
        </div>
         
        <div
          class="access-data"
        >
          <h2>
            Registers:
          </h2>
           
          <h3>
            0
          </h3>
        </div>
      </div>
    </div>
  </div>
</div>
`;
